torch/csrc/distributed/c10d/init.cpp

```diff
@@ -459,6 +459,9 @@ An enum-like class for built-in communication hooks: ``ALLREDUCE`` and ``FP16_CO
             return reducer.get_grad_buckets(/* return_zero_tensors */ true);
           },
           py::call_guard<py::gil_scoped_release>())
+      .def("_set_grads_to_none", [](::c10d::Reducer& reducer) {
+          reducer.set_grads_to_none(true);
```

Comment:
Conceptual: What does the `true` in `reduce.set_grads_to_none(true)` mean?

Reply:
It means that we should set the grads to None.